### PR TITLE
docs: improvement for important docs

### PR DESCRIPTION
**Pull request by lumianph**
For the standalone quickstart, I add the index for c1 when creating the table, to simplify the description of performance-sensitive mode in this quickstart, please check @dl239

### CODE REVIEW

**Comment by lumianph** [comment]
[docs reorganize, review and udpate](https://trello.com/c/EW0F7yOb/1-docs-reorganize-review-and-udpate)

File: docs/en/standalone.md [hunk 1]
```diff
@@ -1,138 +1,98 @@
...
+> CREATE TABLE demo_table1(c1 string, c2 int, c3 bigint, c4 float, c5 double, c6 timestamp, c7 date, INDEX(key=c1, ts=c6));
 ```
-**Note**: Specify at least one index and set the `ts` column which is used for ORDERBY. The `ts` column is the key in `index` option and can be setted with `timestamp` or `bigint` column only. 
-### Import Data
-Only support csv file format (a sample csv file can be downloaded [here](../../demo/standalone/data/data.csv))
+The `INDEX` function used in `CREATE TABLE` accepts two important parameters `key` and `ts`. 
+
+- `key` specifies the index column. If it is not specified, OpenMLDB will use the first applicable column as the index column automatically. Necessary indexes will be built when deploying online SQL. 
+- `ts` specifies the ordered column, which is used by `ORDER BY`. Only the `timestamp` and `bigint` columns can be specified as `ts`. 
+- `ts` and `key` also determine whether certain SQL queries can be executed offline, please refer to [Introduction to OpenMLDB's Performance-Sensitive Mode](performance_sensitive_mode.md) for more details.
```
Comment (dl239) [inline docs/en/standalone.md:48]: `can be executed offline` -> `can be executed online`

File: docs/en/standalone.md [hunk 2]
```diff
@@ -1,138 +1,98 @@
...
+
```
Comment (dl239) [inline docs/en/standalone.md:49]: TTL can be specified  in `INDEX` function.  The conf of TTL contains `ttl_type` and ttl value.  OpenMLDB has four TTL strategies/type (absolute, latest, absandlat, absorlat). We can set ttl as follows:
INDEX(key=c1, ts=c6, ttl=100m, ttl_type=absolute)
INDEX(key=c1, ts=c6, ttl=10, ttl_type=latest)
INDEX(key=c1, ts=c6, ttl=(120min, 100), ttl_type=absandlat)
INDEX(key=c1, ts=c6, ttl=(120min, 100), ttl_type=absorlat)
The default ttl_type is `absolute`, default ttl is 0

Comment (lumianph) [reply docs/en/standalone.md:49]: lets move those detail to a user guide doc, not in the quickstart

File: demo/README.md [hunk 3]
```diff
@@ -1,134 +1,154 @@
...
+> CREATE TABLE t1(id string, vendor_id int, pickup_datetime timestamp, dropoff_datetime timestamp, passenger_count int, pickup_longitude double, pickup_latitude double, dropoff_longitude double,dropoff_latitude double, store_and_fwd_flag string,trip_duration int, INDEX(ts=pickup_datetime));
 > LOAD DATA INFILE './data/taxi_tour.csv' INTO TABLE t1;
 ```
-### Run feature extraction
-```
+**Run offline feature extraction**
+
+```sql
+# The below commands are executed in the CLI
 > SET PERFORMANCE_SENSITIVE = false;
```
Comment (dl239) [inline demo/README.md:83]: `SET PERFORMANCE_SENSITIVE` has been deleted in standalone quickstart.  Should keep consistent with quickstart?

File: docs/en/cluster.md [hunk 4]
```diff
@@ -1,44 +1,33 @@
 
-# QuickStart (Cluster Version)
+# OpenMLDB QuickStart (Cluster Mode)
 
-This tutorial is targeted at the cluster version of OpenMLDB and it will cover the whole lifecycle of how to build a machine learning application with the help of OpenMLDB,
-including feature extraction, model training; online data import, online feature extraction, model prediction, etc.
-From this tutorial, readers can understand how to use OpenMLDB to complete the machine learning lifecycle from raw data to model deployment.
+This tutorial is targeted at the cluster mode of OpenMLDB and it will cover the whole lifecycle of how to build a machine learning application with the help of OpenMLDB, including feature extraction, model training, online data import, online feature extraction, model prediction, etc. From this tutorial, readers can understand how to use OpenMLDB to complete the machine learning lifecycle from raw data to model deployment. OpenMLDB provides both Java and Python SDKs. In this tutorial, we will Python SDK.
```
Comment (dl239) [inline docs/en/cluster.md]: `we will Python SDK` should add `use` after `will` ?

File: demo/README.md (hunk 3 above)
Comment (lumianph) [reply demo/README.md:83]: yes, i intend to delete, but we also should change the `create table`?

Comment (dl239) [reply demo/README.md:83]: yes

**Review by zhanghaohit — APPROVED** [review]
LGTM

File: demo/README.md [hunk 5]
```diff
@@ -1,134 +1,153 @@
...
+avg(pickup_latitude) OVER w AS vendor_avg_pl,
+sum(pickup_latitude) OVER w2 AS pc_sum_pl,
+max(pickup_latitude) OVER w2 AS pc_max_pl,
+min(pickup_latitude) OVER w2 AS pc_min_pl,
+avg(pickup_latitude) OVER w2 AS pc_avg_pl,
+count(vendor_id) OVER w2 AS pc_cnt,
+count(vendor_id) OVER w AS vendor_cnt
+FROM t1
+WINDOW w AS (PARTITION BY vendor_id ORDER BY pickup_datetime ROWS_RANGE BETWEEN 1d PRECEDING AND CURRENT ROW),
+w2 AS (PARTITION BY passenger_count ORDER BY pickup_datetime ROWS_RANGE BETWEEN 1d PRECEDING AND CURRENT ROW) INTO OUTFILE '/tmp/feature.csv';
```
Comment (dl239) [inline demo/README.md:97]: This SQL execute failed if not set PERFORMANCE_SENSITIVE to false now.  https://github.com/4paradigm/OpenMLDB/issues/806

**Review by tobegit3hub — APPROVED** [review]
LGTM

File: demo/README.md (hunk 5 above)
Comment (lumianph) [reply demo/README.md:97]: okay, let's fix this bug first then merge this PR

Comment (lumianph) [reply demo/README.md:97]: i have restored the performance_sensitive=false

**Comment by lumianph** [comment]
[docs reorganize, review and udpate](https://trello.com/c/EW0F7yOb/1-docs-reorganize-review-and-udpate)

**Pull request merged**